src/browser-sandbox/pruneNonCriticalSelectors.js

```diff
@@ -20,14 +20,22 @@ export default function pruneNonCriticalSelectors ({
     // temporarily force clear none in order to catch elements that clear previous
     // content themselves and who w/o their styles could show up unstyled in above
     // the fold content (if they rely on f.e. 'clear:both;' to clear some main content)
-    var originalClearStyle = element.style.clear || ''
-    element.style.clear = 'none'
+
+    // but do that only if elements have their style attribute defined
+    // (ref: https://github.com/pocketjoso/penthouse/issues/342)
+    if (typeof element.style !== 'undefined') {
```

Comment:
to save some computation since this code runs on a hot cod path inside a loop (once for each selector in the css) - can you assign this typeof check to a variable and re-use in the second if condition (on line 35)?

Reply:
Sure!